### PR TITLE
Rename method arguments to use camelCase casing (component directories)

### PR DESCRIPTION
**Pull request by SharkyKZ**
Partial Pull Request for Issue https://github.com/joomla/joomla-cms/issues/30988.

### Summary of Changes

Renames some method arguments to use camelCase casing.

### Testing Instructions

Thorough code review / Joomla works like before.

### Documentation Changes Required

No.

### CODE REVIEW

File: administrator/components/com_users/models/user.php [hunk 1]
```diff
@@ -1210,20 +1210,20 @@ public function generateOteps($user_id, $count = 10)
 	 *					authentication method enabled.
 	 * warn_irq_msg		The string to use for the warn_if_not_req warning
 	 *
-	 * @param   integer  $user_id    The user's numeric ID
+	 * @param   integer  $userId     The user's numeric ID
 	 * @param   string   $secretkey  The secret key you want to check
```
Comment (Quy) [inline administrator/components/com_users/models/user.php]: `secretKey`?

Comment (zero-24) [reply administrator/components/com_users/models/user.php]: Agree. Seems I'm not allowed to make suggestions for all lines so I will let that up to you.

Comment (SharkyKZ) [reply administrator/components/com_users/models/user.php]: Renamed.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on cd37e86458220960ac5654334e4e8e2cab3ce25a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31154">issues.joomla.org/tracker/joomla-cms/31154</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on cd37e86458220960ac5654334e4e8e2cab3ce25a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31154">issues.joomla.org/tracker/joomla-cms/31154</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31154">issues.joomla.org/tracker/joomla-cms/31154</a>.</sub>

**Pull request merged**